src/renderer/modules/webpack.ts

```diff
@@ -340,3 +301,101 @@ export function patchPlaintext(patches: PlaintextPatch[]): void {
     })),
   );
 }
+
+// Helpers for the lazy
+
+export function getBySource(
+  match: string | RegExp,
+  options?: { all?: false; raw?: false },
+): ModuleExports | undefined;
+export function getBySource(
+  match: string | RegExp,
+  options?: { all?: true; raw?: false },
+): ModuleExports[];
+export function getBySource(
+  match: string | RegExp,
+  options?: { all?: false; raw?: true },
+): RawModule | undefined;
+export function getBySource(
+  match: string | RegExp,
+  options?: { all?: true; raw?: true },
+): RawModule[];
+
+export function getBySource(
+  match: string | RegExp,
+  options: GetModuleOptions | undefined = {
+    all: false,
+    raw: false,
+  },
+): ModuleExports[] | RawModule[] | ModuleExports | RawModule | undefined {
+  // @ts-expect-error https://github.com/microsoft/TypeScript/issues/26242
+  return getModule(bySource(match), options);
+}
+
+export function getByProps<P extends string>(
+  props: P[],
+  options: { all?: false; raw?: false },
+): ModuleExportsWithProps<P> | undefined;
+export function getByProps<P extends string>(
+  props: P[],
+  options: { all?: true; raw?: false },
+): Array<ModuleExportsWithProps<P>>;
+export function getByProps<P extends string>(
+  props: P[],
+  options: { all?: false; raw?: true },
+): RawModule | undefined;
+export function getByProps<P extends string>(
+  props: P[],
+  options: { all?: true; raw?: true },
+): RawModule[];
+export function getByProps<P extends string>(...props: P[]): ModuleExportsWithProps<P> | undefined;
+
+export function getByProps<P extends string>(
+  ...args: [P[], GetModuleOptions] | P[]
+):
```

Comment:
This mess of TS makes me wince so hard but there's not much that can be done about it without separate functions like we had before